packages/apollo-gateway/src/index.ts

```diff
@@ -101,7 +130,7 @@ export class ApolloGateway implements GraphQLService {
   protected createSchema(services: ServiceDefinition[]) {
     this.logger.debug(
       `Composing schema from service list: \n${services
-        .map(({ name }) => `  ${name}`)
+        .map(({ name, url }) => `  ${url || 'local'} : ${name}`)
```

Comment:
The space between these looks weird when I imagine this actual message in practice?

```suggestion
        .map(({ name, url }) => `  ${url || 'local'}: ${name}`)
```

Reply:
Put it like that so you wouldn't have 

`localhost:4004: accounts`
`localhost:4004 : accounts`

idk